reference/json/BigCommerce_Process_Payment_API.oas2.json

```diff
@@ -652,8 +652,27 @@
         "title",
         "type"
       ]
-    }
-  },
+    },
+  "StoredPayPalAccount": {
+    "title": "Stored Paypal Account",
+    "type": "object",
+    "properties": {
+      "type": {
```

Comment:
The `type` property here needs to be set as an enum with `stored_paypal_account` as the only option: 

```yaml
    StoredPayPalAccount:
      type: object
      properties:
        type:
          type: string
          description: "Type to classify this payment instrument"
          enum:
            - stored_paypal_account
```